products/lucene.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Apache Lucene
+category: library
+tags: apache java-runtime
+iconSlug: apache
+permalink: /lucene
+alternate_urls:
+-   /apache-lucene
+releasePolicyLink: https://lucene.apache.org/core/downloads.html
+changelogTemplate: "https://lucene.apache.org/core/{{'__LATEST__'|replace:'.','_'}}/changes/Changes.html"
+releaseDateColumn: true
+
+auto:
+  methods:
+  -   git: https://github.com/apache/lucene.git
+      regex: '^releases\/lucene\/(?P<version>\d+\.\d+(.\d+)?)$'
```

Comment:
The 7.x and 8.x releases are missing unless we include the `lucene-solr` releases. https://regex101.com/r/l3usV9/1

```suggestion
      regex: '^releases\/lucene(-solr)?\/(?P<version>\d+\.\d+(.\d+)?)$'
```